frontend/src/containers/RecipePage/DeleteIcon/DeleteIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './DeleteIcon.css';
+
+const DeleteIcon = () => (
```

Comment:
Proper camelCasing for variable names usually starts off with a lowercase letter. So instead, i would make 'DeleteIcon' into 'deleteIcon'